### PR TITLE
💄fixed special characters file (now readable and in markdown)

### PR DESCRIPTION
**Pull request by mkruijt**

### CODE REVIEW

File: fundamentals/names_of_special_characters.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+Do you think there is one (or more) missing? Make a PR with your additions 🙃!
+
+
+`(` Parentheses `)`
+
+`{` Braces, or curly braces `}`
+
+`[` Brackets, or square brackets `]`
+
+`<` Angeled brackets `>`
```
[the remark this answers is not in the document]
Comment (mkruijt) [reply fundamentals/names_of_special_characters.md]: https://en.wiktionary.org/wiki/angle_bracket

File: fundamentals/names_of_special_characters.md [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
+`/` Forward slash `/`
+
+`\` Back slash `\`
+
+`:` Colon `:`
+
+`;` Semicolon `;`
+
+`...` The spread operator, also called Ellipses
+
```
[the remark this answers is not in the document]
Comment (mkruijt) [reply fundamentals/names_of_special_characters.md]: yes I agree, I will take this one out

File: fundamentals/names_of_special_characters.md [hunk 3]
```diff
@@ -1,21 +1,43 @@
...
+`*` Asterisk `*`
+
+`^` Caret or circumflex `^`
+
+`|` Vertical bar `|`
+
+`~` Tilde or equivalency sign `~`
+
+`#` Hash or number sign `#`
```
[the remark this answers is not in the document]
Comment (mkruijt) [reply fundamentals/names_of_special_characters.md]: yes nice suggestion

File: fundamentals/names_of_special_characters.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (remarcmij) [reply fundamentals/names_of_special_characters.md]: I stand corrected. I have always called them angular brackets but Google proves me wrong 😄.

[the remark this answers is not in the document]
Comment (mkruijt) [reply fundamentals/names_of_special_characters.md]: well the plural form of angle is actually angle, so there is a mistake in there 🙃

**Comment by mkruijt** [comment]
@remarcmij what do you think about calling the tilde also "equivalent sign"? I assume that is what Sam meant with "equivalency sign" right @hackoor83 ? It is more common to use ≈ for that.

**Comment by remarcmij** [comment]
Actually, I would just call it a tilde. As operator in JS it is not often used (Bitwise NOT, I had to look it up in MDN). I think it is more often encountered as representing the home directory in Unix.

**Comment by mkruijt** [comment]
💃

**Comment by remarcmij** [comment]
BTW, from where to we link to it? Fundamentals main page, operators page?

**Pull request merged**